addons/repair/migrations/13.0.1.0/openupgrade_analysis.txt

```diff
@@ -1,16 +1,20 @@
 ---Models in module 'repair'---
 new model repair.tags
+# NOTHING TO DO
 ---Fields in module 'repair'---
 repair       / account.move             / repair_ids (one2many)         : NEW relation: repair.order
 repair       / account.move.line        / repair_fee_ids (one2many)     : NEW relation: repair.fee
 repair       / account.move.line        / repair_line_ids (one2many)    : NEW relation: repair.line
+# NOTHING TO DO
 repair       / repair.fee               / invoice_line_id (many2one)    : relation is now 'account.move.line' ('account.invoice.line') [nothing to do]
 repair       / repair.line              / invoice_line_id (many2one)    : relation is now 'account.move.line' ('account.invoice.line') [nothing to do]
 repair       / repair.order             / invoice_id (many2one)         : relation is now 'account.move' ('account.invoice') [nothing to do]
+# DONE: handled the new relation
 repair       / repair.order             / tag_ids (many2many)           : NEW relation: repair.tags
 repair       / repair.order             / user_id (many2one)            : NEW relation: res.users, hasdefault
```

Comment:
Is the provided default correct? Probably not, and you need to use the create_user for example.

Reply:
This comment is still pending